### ISSUE TITLE
Build instructions for spawnedshelter

### ISSUE THREAD

**Issue by kvakvs** [unbalancedparentheses/spawnedshelter#32]
Please add a paragraph to README how to install (something-something npm install didn't really succeed for me first with missing packages then with some exceptions which i would not care to investigate) and how to make the website. For javascript-challenged minds. We do Erlang, not Javascript.

In short: I installed node, installed stylus for npm, installed pandoc, installed god knows what else (sudo npm install -g), and it gave me postcss error Promise not defined. That is not something i want to fix. Did I install it wrong? A wrong node maybe.

Perfectly a single run of `make` should have done all preparations.

**Comment by unbalancedparentheses** [comment]
@kvakvs thanks for submiting the issue. Las week I created a task for this #19. I did not have time to do it yet. 

make deps should be enough to get the dependencies if you have node and haskell (needed for pandoc) installed. Probably postcss (a library needed for adding css prefixes for each browser) has some  issue with your node version. At the end of the day I will try it and I will add the instructions.
